web/Migrations/BricksAndHeartsDbContextModelSnapshot.cs

```diff
@@ -67,6 +67,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<string>("InviteLink")
                         .HasColumnType("nvarchar(max)");
 
+                    b.Property<bool>("IsAssigned")
+                        .HasColumnType("bit");

```

Comment:
Is this supposed to be here?

Reply:
Not really - I had minor problems getting the filtering to work (I was trying to use the function FindUserByLandlordId inside the landlordQuery.Where()), and at one stage added another field to the database. I then realised how much of a pain that would be and undid the database migrations, but those lines must have stayed around.